### ISSUE TITLE
Netgear a7000 rtl8814au not work

### ISSUE THREAD

**Issue by rohtakcity** [sebanc/brunch#854]
Netgear a7000 rtl8814au not work. Can developer solve thiss issue.

**Comment by sebanc** [comment]
Could you try the build in the "brunch-unstable" repository and add "options=rtl8814au" on the kernel command line ?

**Comment by rohtakcity** [comment]
I try "ottions=rtl8814au" but this one not work.

**Comment by sebanc** [comment]
Did you try the build from the "brunch-unstable" repository on my github ? (not this one)

**Comment by rohtakcity** [comment]
No, could you please guide me how to use "brunch-unstable" repository. I shall be thankful to you if it's work for me

**Comment by sebanc** [comment]
try this build:
https://github.com/sebanc/brunch-unstable/releases/download/r88-unstable-20210207/brunch_r88_unstable_20210207.tar.gz
